source/applications/dependency-injection.md

```diff
@@ -209,3 +215,28 @@ export default {
   initialize: initialize
 };
 ```
+
+### Getting an Application Instance from a Factory Instance
+
+[`Ember.getOwner`][4] will retrieve the application instance from an Ember
+Object that was instantiated from a registered factory. This means that
```

Comment:
something like

"Factory instances in Ember are "owned" by the application instance object that created them. The `Ember.getOwner` function will retrieve the application instance object relevant to a factory instance, which may be useful for fetching other instances at runtime."

At the least "Ember Object" should probably not capitalize "Object" as it may imply something special about `Ember.Object`.